src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidInputException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl registator = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUpDefaultUser() {
+        user.setLogin("GhostVoice");
+        user.setPassword("123456");
+        user.setAge(19);
+        Storage.PEOPLE.add(user);
+    }
+
```

Comment:
Do you have test for user with valid name, age and password?)

Reply:
agree